src/ducks/Notifications/NoNotitications/utils.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+export const NoNotificationsSvg = (
+  <svg
+    width='135'
+    height='130'
+    viewBox='0 0 135 130'
+    fill='none'
+    xmlns='http://www.w3.org/2000/svg'
```

Comment:
minify paths, please and remove xmlns attr

Reply:
good suggestion, ty